DMTR-392.tex

```diff
@@ -25,7 +29,7 @@
 % \setDocCurator{The Curator of this Document}
 
 \setDocAbstract{%
-Add abstract text.
+This brief report describes measurements of data quality metrics that were carried out for release v25.0.0 of the LSST Science Pipelines. The report for the previous version can be found in \citedsp{DMTR-391}.
```

Comment:
The citation is not rendering on the webpage

Reply:
Should be fixed now. Needed to add it to lsst-texmf.